### PR TITLE
[12.0][MIG] document_quick_access_folder_auto_classification

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration. Tested and works without major changes.

@etobella @hbrunn @LoisRForgeFlow

### CODE REVIEW

**Review by JoanSForgeFlow — APPROVED** [review]
Tested on local, works great. :+1: 

Runbot only fails because of a missing description.

**Comment by jarroyomorales** [comment]
@JoanSForgeFlow Fixed! Thank you!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM! :+1:

File: document_quick_access_folder_auto_classification/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+{
+    'name': 'Document Quick Access Folder Auto Classification',
+    'summary': """
+        Auto classification of Documents after reading a QR""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Creu Blanca,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/server-ux',
+    'depends': [
```
Comment (LoisRForgeFlow) [inline document_quick_access_folder_auto_classification/__manifest__.py:12]: A small comment, can you add here the python dependencies?

Comment (jarroyomorales) [reply document_quick_access_folder_auto_classification/__manifest__.py:12]: Done!

**Comment by LoisRForgeFlow** [comment]
Thanks!

/ocabot merge

**Pull request merged**